docs/decisions/0047-ocr-engine-selection.md

```diff
@@ -0,0 +1,172 @@
+---
+parent: Decision Records
+nav_order: 47
+---
+# OCR Engine Selection for JabRef
+
+## Context and Problem Statement
+
+JabRef requires an OCR engine to extract text from scanned PDFs and image-based academic documents. Tesseract is currently implemented, but accuracy varies significantly with document quality and type. Academic documents present unique challenges: mathematical notation, multiple languages, complex layouts, tables, and mixed handwritten/printed content. Which OCR engine(s) should JabRef adopt to best serve its academic user base while balancing accuracy, cost, privacy, and implementation complexity?
+
+## Decision Drivers
+
+* Accuracy on academic document types (printed papers, scanned books, handwritten notes)
+* Privacy requirements for unpublished research materials
+* Cost constraints (open-source project with limited funding)
+* Language support for international academic community
+* Support for mathematical and scientific notation
+* Offline capability for secure research environments
+* Processing speed for batch operations
+* Implementation and maintenance complexity
+* Table and structure extraction capabilities
+* Long-term sustainability and community support
+
+## Considered Options
+
+* Option 1: Tesseract OCR (current implementation)
+* Option 2: Google Cloud Vision API
+* Option 3: AWS Textract
+* Option 4: Microsoft Azure Computer Vision
+* Option 5: EasyOCR
+* Option 6: PaddleOCR
+* Option 7: ABBYY FineReader SDK
+
+### Pros and Cons of the Options
+
+#### Option 1: Tesseract OCR
+
+Originally developed by Hewlett‑Packard as proprietary software in the 1980s, released as open source in 2005 and development was sponsored by Google in 2006.
+
+* **Good**, because Version 4 adds LSTM‑based OCR engine and models for many additional languages and scripts, bringing the total to 116 languages. Additionally 37 scripts are supported
+* **Good**, because completely free and open‑source (Apache 2.0 license)
+* **Good**, because runs entirely offline preserving document privacy
+* **Good**, because has an active developer community that regularly updates it, fixes bugs, and improves performance based on user feedback
+* **Good**, because supports over 100 languages out of the box and can be trained to recognize additional languages or custom fonts
```

Comment:
This text makes the impression that is not hand-crafted.

Arguments are not semantically grouped together - and merged for the same arguments.

"over 100 languages" are is the same as "116 languages"